apps/test/unit/templates/sectionProgress/progressTables/ProgressTableLevelIconTest.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import {expect} from '../../../../util/reconfiguredChai';
+import {shallow} from 'enzyme';
+import ProgressTableLevelIcon from '@cdo/apps/templates/sectionProgress/progressTables/ProgressTableLevelIcon';
+import FontAwesome from '@cdo/apps/templates/FontAwesome';
+
+const DEFAULT_PROPS = {
+  levels: [
+    {
+      id: 123,
+      levelNumber: 1,
+      sublevels: [{id: 1}, {id: 2}],
+      kind: 'assessment'
+    },
+    {
+      id: 456,
+      levelNumber: 2,
+      sublevels: undefined,
+      isUnplugged: true
+    },
+    {
+      id: 789,
+      levelNumber: 3,
+      sublevels: undefined,
+      bonus: true
+    }
+  ]
+};
+
+describe('ProgressTableLevelIcon', () => {
+  it('renders icon for each level', () => {
+    const wrapper = shallow(<ProgressTableLevelIcon {...DEFAULT_PROPS} />);
+    expect(wrapper.find(FontAwesome)).to.have.length(3);
+  });
+
+  it('renders extra space for sublevels', () => {
+    const wrapper = shallow(<ProgressTableLevelIcon {...DEFAULT_PROPS} />);
+    const levelWithSublevels = wrapper.findWhere(
+      node => node.key() === '123_1'
+    );
+    expect(levelWithSublevels.childAt(1).props().style.width).to.equal(46);
```

Comment:
This test seems oddly specific (width should be exactly 46 pixels), but not sure if there's a better way to express the behavior we're trying to verify.

Reply:
I could add a class name or factor out the span to a separate component, and then use that to check that it's present in the unit test

Reply:
<shrug> seems like overkill?  Maybe reference the constant that equals 46 or at least add a comment with the constant name so it will come up when grepping the codebase?